src/components/FullPageSpinner/FullPageSpinner.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import Loading from 'components/Loading';
+
+import styles from './FullPageSpinner.module.scss';
+
+const FullPageSpinner = () => (
```

Comment:
what do you think about making this a "Page"?

Reply:
I don't find precise, we always render this component as a content loader, isn't a page itself. Maybe the current name isn't accurate, I could rename it to `FullSizeSpinner`, thoughts?

Reply:
why do you think it's not a page itself? It's actually being render instead of the pages while they are loading. I think that the content loader you are referring to is the `Loading` component

Reply:
Sorry I got confused, you're right, I'll change it 👍